lang/fpc/Portfile

```diff
@@ -263,7 +263,7 @@ if {${subport} eq "${name}"} {
 
     if       {${os.major} >= 11}   { # Big Sur
         set otherOptions "-WM11.0 -XR${configure.sdkroot}"
-    } elseif {[vercmp ${os.version} 10.9] >= 0} { # Mavericks
+    } elseif {[vercmp ${macos_version} 10.9] >= 0} { # Mavericks
```

Comment:
As I said in another comment, I recommend checking `os.major` instead, and then you won't need to use `vercmp`.

However, if `vercmp` needs to be used, `[vercmp ${macos_version} 10.9] >= 0` the old complicated syntax. The newer less-complicated syntax is `[vercmp ${macos_version} >= 10.9]`

Reply:
oh. Very nice. Thanks.